curriculum/challenges/english/07-scientific-computing-with-python/learn-classes-and-objects-by-building-a-sudoku-solver/6569f70a66ccdc42097ca051.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-64
 
 # --description--
 
-If still there are empty cells, create a loop in the `else` block that iterates over numbers from `1` to `9` (inclusive).
+If there are still empty cells, create a loop in the `else` block. Use `guess` to iterate over numbers from `1` to `9` (inclusive).
```

Comment:
To me it sounds like the current wording suggests that the variable `guess` is being used to iterate over the numbers when the loop is doing that.

maybe something like this would be clearer

```suggestion
If there are still empty cells, create a loop in the `else` block that iterates over numbers from `1` to `9` (inclusive). Your loop should use the variable named `guess`. 
```

Reply:
That is worded in a better manner than my suggestion. Thanks.